core/src/main/java/org/dst/core/operatorImpl/DstListImpl.java

```diff
@@ -1,55 +1,95 @@
 package org.dst.core.operatorImpl;
 
-import org.dst.core.exception.NotImplementException;
 import org.dst.core.operatorset.DstList;
+import org.dst.utils.enums.Status;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.HashMap;
 import java.util.List;
 
 public class DstListImpl implements DstList {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(DstListImpl.class);
+
   private HashMap<String, List<String>> listMap;
 
   public DstListImpl() {
     this.listMap = new HashMap<String, List<String>>();
   }
 
   @Override
-  public void put(String key, List<String> value) {
+  public Status put(String key, List<String> value) {
     listMap.put(key, value);
+    return Status.OK;
   }
 
   @Override
   public List<String> get(String key) {
+    if (!listMap.containsKey(key)) {
+      //TODO should return a status value
```

Comment:
Maybe we can simply to return a null here?

Reply:
got it!